lib/realtime/monitoring/prom_ex/plugins/tenant.ex

```diff
@@ -53,11 +60,12 @@ defmodule Realtime.PromEx.Plugins.Tenant do
 
     for t <- tenants do
       count = UsersCounter.tenant_users(Node.self(), t)
+      cluster_count = UsersCounter.tenant_users(t)
```

Comment:
This value will be the same on all nodes and each node logs it. Is what you expect?

Reply:
Yep!